language/German/strings.po

```diff
@@ -3970,575 +4031,575 @@ msgstr ""
 msgid "2 Players Start"
-msgstr ""
+msgstr "2 Spieler Start"
 
 #: src/emu/inpttype.ipp:561
 msgctxt "input-name"
 msgid "3 Players Start"
-msgstr ""
+msgstr "3 Spieler Start"
 
 #: src/emu/inpttype.ipp:562
 msgctxt "input-name"
 msgid "4 Players Start"
-msgstr ""
+msgstr "4 Spieler Start"
 
 #: src/emu/inpttype.ipp:563
 msgctxt "input-name"
 msgid "5 Players Start"
-msgstr ""
+msgstr "5 Spieler Start"
 
 #: src/emu/inpttype.ipp:564
 msgctxt "input-name"
 msgid "6 Players Start"
-msgstr ""
+msgstr "6 Spieler Start"
 
 #: src/emu/inpttype.ipp:565
 msgctxt "input-name"
 msgid "7 Players Start"
-msgstr ""
+msgstr "7 Spieler Start"
 
 #: src/emu/inpttype.ipp:566
 msgctxt "input-name"
 msgid "8 Players Start"
-msgstr ""
+msgstr "8 Spieler Start"
 
 #: src/emu/inpttype.ipp:571
 msgctxt "input-name"
 msgid "Coin 1"
-msgstr ""
+msgstr "Münze 1"
 
 #: src/emu/inpttype.ipp:572
 msgctxt "input-name"
 msgid "Coin 2"
-msgstr ""
+msgstr "Münze 2"
 
 #: src/emu/inpttype.ipp:573
 msgctxt "input-name"
 msgid "Coin 3"
-msgstr ""
+msgstr "Münze 3"
 
 #: src/emu/inpttype.ipp:574
 msgctxt "input-name"
 msgid "Coin 4"
-msgstr ""
+msgstr "Münze 4"
 
 #: src/emu/inpttype.ipp:575
 msgctxt "input-name"
 msgid "Coin 5"
-msgstr ""
+msgstr "Münze 5"
 
 #: src/emu/inpttype.ipp:576
 msgctxt "input-name"
 msgid "Coin 6"
-msgstr ""
+msgstr "Münze 6"
 
 #: src/emu/inpttype.ipp:577
 msgctxt "input-name"
 msgid "Coin 7"
-msgstr ""
+msgstr "Münze 7"
 
 #: src/emu/inpttype.ipp:578
 msgctxt "input-name"
 msgid "Coin 8"
-msgstr ""
+msgstr "Münze 8"
 
 #: src/emu/inpttype.ipp:579
 msgctxt "input-name"
 msgid "Coin 9"
-msgstr ""
+msgstr "Münze 9"
 
 #: src/emu/inpttype.ipp:580
 msgctxt "input-name"
 msgid "Coin 10"
-msgstr ""
+msgstr "Münze 10"
 
 #: src/emu/inpttype.ipp:581
 msgctxt "input-name"
 msgid "Coin 11"
-msgstr ""
+msgstr "Münze 11"
 
 #: src/emu/inpttype.ipp:582
 msgctxt "input-name"
 msgid "Coin 12"
-msgstr ""
+msgstr "Münze 12"
 
 #: src/emu/inpttype.ipp:583
 msgctxt "input-name"
 msgid "Bill 1"
-msgstr ""
+msgstr "Geldschein 1"
 
 #: src/emu/inpttype.ipp:588
 msgctxt "input-name"
 msgid "Service 1"
-msgstr ""
+msgstr "Service 1"
 
 #: src/emu/inpttype.ipp:589
 msgctxt "input-name"
 msgid "Service 2"
-msgstr ""
+msgstr "Service 2"
 
 #: src/emu/inpttype.ipp:590
 msgctxt "input-name"
 msgid "Service 3"
-msgstr ""
+msgstr "Service 3"
 
 #: src/emu/inpttype.ipp:591
 msgctxt "input-name"
 msgid "Service 4"
-msgstr ""
+msgstr "Service 4"
 
 #: src/emu/inpttype.ipp:596
 msgctxt "input-name"
 msgid "Tilt 1"
-msgstr ""
+msgstr "Neigungsschalter 1"
 
 #: src/emu/inpttype.ipp:597
 msgctxt "input-name"
 msgid "Tilt 2"
-msgstr ""
+msgstr "Neigungsschalter 2"
 
 #: src/emu/inpttype.ipp:598
 msgctxt "input-name"
 msgid "Tilt 3"
-msgstr ""
+msgstr "Neigungsschalter 3"
 
 #: src/emu/inpttype.ipp:599
 msgctxt "input-name"
 msgid "Tilt 4"
-msgstr ""
+msgstr "Neigungsschalter 4"
 
 #: src/emu/inpttype.ipp:604
 msgctxt "input-name"
 msgid "Power On"
-msgstr ""
+msgstr "Anschalter"
```

Comment:
"Einschalten"?

Reply:
I thought this is naming the inputs, in which case its the physical button?

Reply:
yeah, it's okay as-is, I think "Anschalten" is fine too

Reply:
Yes, it’s the power on button.